src/test/resources/log4j2.properties

```diff
@@ -0,0 +1,10 @@
+# Logging settings for unit tests
+
+# The root logger with appender name
+rootLogger=DEBUG, STDOUT
+
+# Assign STDOUT a valid appender & define its layout
+appender.console.name=STDOUT
+appender.console.type=Console
+appender.console.layout.type=PatternLayout
+appender.console.layout.pattern=%msg%n
```

Comment:
```patch
- appender.console.layout.pattern=%msg%n
+ appender.console.layout.pattern=%d %-5p %c{1} %m%n
```

Reply:
Maven like formatting (colors, layout) would be more like
```
appender.console.layout.pattern=[%highlight{%p}{INFO=blue}]%m%n
```
or 
```
appender.console.layout.pattern=[%highlight{%p}{INFO=blue}] [%c{1}] %m%n
```

Reply:
I went with the latest proposal.